### DISCUSSION TITLE
Pen pressure and tablet buttons in MacOS

### DISCUSSION THREAD

**Discussion by plasticfrank** [OpenTabletDriver/OpenTabletDriver#1461] [Q&A] [answered]
MacOS 10.13.6
Wacom CTL-480

Stylus works but no pressure sensitivity or button function.

OTD works better than the Wacom drivers but how do I make pressure sensitivity.

Documentation seems either lacking context or over my head. I have no idea what I'm looking at. 

Is it even possible to get more out of the (experimental) Mac version. 

Also, where do I donate to the project? I'd like to see OTD polished, a little more user friendly and supported for a long time to come. People shouldn't be forced to bin perfectly functional tablets.

**Comment by adryzz** [comment, marked as answer]
Pressure support on MacOS is lacking, as MacOS doesn't allow to control the pressure in usermode, but only to move the cursor.

**Reply by plasticfrank** [reply]
I know this is outside the topic of this forum but any ideas wear I go from here?

I turned to OpenTabletDriver because the Wacom driver interferes with keyboard shortcuts in Photoshop CS6 (not Krita for some reason). 

OpenTabletDriver works for some applications. But for some purposes I actually need pressure sensitivity. I have two different Wacoms affected by this. Any ideas?

**Reply by adryzz** [reply]
The only options are either use the Wacom drivers or don't use macOS (or write a driver that isn't user mode, make Apple sign it and install it, but that is never gonna happen)

**Reply by HybridEidolon** [reply]
@adryzz Are you certain that's true? The February 2021 HuionTablet.app for the Huion NEW 1060 Plus (8192) is able to provide pressure data to CSP without installing any driver on Monterey 12.4. I couldn't tell you _how_ it's doing it, I'm not very familiar with mac app dev, but it is.

[HuionTablet_MacDriver_v14.6.0.210128.zip](https://driverdl.huion.com/driver/Mac/HuionTablet_MacDriver_v14.6.0.210128.zip)

**Comment by thenickdude** [comment]
Wacom's official driver is a user-mode app, you don't need a kernel driver in order to deliver pressure on macOS.

**Reply by jamesbt365** [reply]
Given that you've commented on a nearly 2 year old discussion stuff has changed and information is outdated. See #2370 and #2371.
